public/opensource-projects/index.json

```diff
@@ -0,0 +1,255 @@
+    "issue": "https://github.com/zkSNACKs/WalletWasabi/issues"
+  },
+  "fedimint": {
+    "repo_url": "https://github.com/fedimint/fedimint",
+    "name": "fedimint",
+    "org": "fedimint",
+    "tags": ["lightning", "bitcoin", "p2p", "lightning-network", "e-cash", "gateways", "protocol"],
+    "lang": ["Rust", "Shell", "Nix"],
+    "issue": "https://github.com/fedimint/fedimint/issues"
+  },
+  "ui": {
+    "repo_url": "https://github.com/fedimint/ui",
+    "name": "ui",
+    "org": "fedimint",
+    "tags": ["lightning", "bitcoin", "p2p", "lightning-network", "e-cash", "gateways"],
+    "lang": ["Typescript"],
+    "issue": "https://github.com/fedimint/ui/issues"
+  },
+  "polar": {
+    "repo_url": "https://github.com/jamaljsr/polar",
+    "name": "polar",
+    "org": "",
+    "tags": ["lightning", "bitcoin", "lightning-network", "polar", "bitcoin-d", "c-lightning"],
+    "lang": ["Typescript", "Docker"],
+    "issue": "https://github.com/jamaljsr/polar/issues"
+  },
+  "bitcoin": {
+    "repo_url": "https://github.com/bitcoin/bitcoin",
+    "name": "bitcoin",
+    "org": "Bitcoin",
+    "tags": ["cryptography", "bitcoin", "p2p", "cryptocurrency", "blockchain", "protocol"],
+    "lang": ["C++", "Python", "C"],
+    "issue": "https://github.com/bitcoin/bitcoin/issues"
+  },
+  "xpub-converter": {
+    "repo_url": "https://github.com/jlopp/xpub-converter",
+    "name": "xpub-converter",
+    "org": "jlopp",
+    "tags": ["bitcoin"],
+    "lang": ["Javascript"],
+    "issue": "https://github.com/jlopp/xpub-converter/issues"
+  },
+  "bitcoinjs-lib": {
+    "repo_url": "https://github.com/bitcoinjs/bitcoinjs-lib",
+    "name": "bitcoinjs-lib",
+    "org": "bitcoinjs",
+    "tags": ["bitcoin", "bitcoinjs", "bitcoinjs-lib", "library"],
+    "lang": ["Typescript", "Javascript"],
+    "issue": "https://github.com/bitcoinjs/bitcoinjs-lib/issues"
+  },
+  "electrum": {
+    "repo_url": "https://github.com/spesmilo/electrum",
+    "name": "electrum",
+    "org": "spesmilo",
+    "tags": ["lightning", "bitcoin", "wallet", "bitcoin-wallet", "lightning-network"],
+    "lang": ["Python", "QML"],
+    "issue": "https://github.com/spesmilo/electrum/issues"
+  },
+  "bluewallet": {
+    "repo_url": "https://github.com/bluewallet/bluewallet",
+    "name": "bluewallet",
+    "org": "BlueWallet",
+    "tags": ["bitcoin", "blockchain", "cryptocurrency", "bitcoinjs", "electrum", "wallet"],
+    "lang": ["Javascript", "Typescript"],
+    "issue": "https://github.com/bluewallet/bluewallet/issues"
+  },
+  "LndHub": {
+    "repo_url": "https://github.com/BlueWallet/LndHub",
+    "name": "LndHub",
+    "org": "BlueWallet",
+    "tags": ["bitcoin", "lightning", "lightning-network"],
+    "lang": ["Javascript"],
+    "issue": "https://github.com/BlueWallet/LndHub/issues"
+  },
+  "rn-ldk": {
+    "repo_url": "https://github.com/BlueWallet/rn-ldk",
+    "name": "rn-ldk",
+    "org": "BlueWallet",
+    "tags": ["bitcoin", "lightning", "lightning-network", "react-native"],
```

Comment:
I think react native should fall under lang? or maybe not include it at all? Or we could just add a new tag called "mobile". this would catch all mobile wallets like bluewallet, phoenix, etc

Reply:
Great suggestion

Reply:
I think react native is fine under tag, we could also add mobile to the tag too

Reply:
@BalogunofAfrica adding react native under tags begs the question of if we should add react, vuejs (which are frameworks not languages by the way) for other projects that uses typescript but in react/nextjs.
Is there a reason you advocate for adding it?

Reply:
@Extheoisah 

My reason is say for a developer who's perhaps a mobile developer (primarily with react native) stumbling upon projects (through filters) in react native that they can immediately be productive. 

I think tags should add these other context that language alone can not provide.